istio/sample_apps.go

```diff
@@ -39,27 +41,43 @@ func (istio *Istio) patchWithEnvoyFilter(namespace string, del bool, app string,
 	if err != nil {
 		return st, ErrEnvoyFilter(err)
 	}
+	var wg sync.WaitGroup
+	var errs []error
+	for _, k8sconfig := range kubeconfigs {
+		wg.Add(1)
```

Comment:
No `wg.Done()` for this

Reply:
Geez, this same code was being copy pasted so many times that I am doing these blunders. Thanks for pointing it out. Please ping if you see any of these